bezier-rs/lib/src/lib.rs

```diff
@@ -729,8 +751,57 @@ impl Bezier {
 			return curve.intersections(self, Some(error));
 		}
 
+		// TODO: Considering using the `intersections_between_vectors_of_curves` helper function here
 		// Otherwise, use bounding box to determine intersections
-		self.intersections_between_subcurves([0., 1.], curve, error)
+		self.intersections_between_subcurves([0., 1.], curve, [0., 1.], error).iter().map(|t_values| t_values[0]).collect()
```

Comment:
Avoid `collect()`, you can just mutate the values provided by `intersections_between_subcurves()`.

Reply:
The return type from `intersections_between_subcurves` is a pair of 2 values. I only want to return one of them here, so I can't mutate the returned list.

Reply:
Ah you're right, I didn't notice that.